### PR TITLE
Prepare for v2.5.0 release.

### PR DESCRIPTION
**Pull request by terrylucas**

### CODE REVIEW

File: tool/pubspec.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: devtools_repo
 description: A repo management tool for DevTools.
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
```
Comment (kenzieschmoll) [inline tool/pubspec.yaml:5]: is this change necessary?

**Review by kenzieschmoll — APPROVED** [review]

File: tool/pubspec.yaml (hunk 1 above)
Comment (terrylucas) [reply tool/pubspec.yaml]: Certainly caused bots to not work.  However, update_version.sh would not run without the sdk change e.g.
```

tool/update_version.dart:50:7: Error: Null safety features are disabled for this library.
Try removing the package language version or setting the language version to 2.12 or higher.
String? incrementVersion(String oldVersion) {
```
